plugins/org/src/components/MySquads/MySquads.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  SidebarItem,
+  SidebarSubmenu,
+  SidebarSubmenuItem,
+} from '@backstage/core-components';
+import {
+  IconComponent,
+  identityApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+
+export const MySquads = ({
```

Comment:
I wonder if this should actually be called `MyGroups` instead? The term `squad` although the right one in many companies, semantically it's `Groups` that we look at in the Catalog?

It's fine to keep the customization so you can display it as `squads` or whatever the company lingo is other than group, but just thinking that it's nicer to use Groups here?

Reply:
Yes, this makes sense to me. I'll work on these changes on Friday when I'll have more time. Thanks!

Reply:
Ok, `MySquads` has been renamed `MyGroups` 👍